packages/help-center/src/components/help-center-content.tsx

```diff
@@ -56,7 +57,13 @@ const HelpCenterContent: React.FC< { isRelative?: boolean; currentRoute?: string
 	const { startNewInteraction } = useManageSupportInteraction();
 	const { data } = useSupportStatus();
 	const { data: openSupportInteraction, isLoading: isLoadingOpenSupportInteractions } =
-		useGetSupportInteractions( 'help-center' );
+		useGetSupportInteractions(
+			'help-center',
+			undefined,
```

Comment:
I would add these undefined values to a descriptive variable name. It makes it easier to read and understand.